### ISSUE TITLE
Screen reader is not announcing the expanded/collapsed state for the dropdown controls present in the header section in Windows.

### ISSUE THREAD

**Issue by msftedad** [wix/react-native-navigation#8106]
### What happened?

**Test Environment:** 
OS: Windows 11 Version 24H2(OS Build 26120.2415)
Browser: Edge Browser 133.0.3065.69 (Official build) (64-bit)
Screen Reader: Narrator.

**User Impact:**
Screen reader users on Windows do not receive feedback on the expanded or collapsed state of dropdown controls in the header section. This lack of announcement can cause confusion, as users may not realize whether a dropdown is open or closed.

**Observation:**
This issue is observed with NVDA and JAWS also.
The dropdown is getting opened automatically when focus moves to the controls.

**Pre-Requisite:**
1. Turn on Screen Reader:
    Narrator: Win + Shift + Enter
2. Verbosity:
    Narrator: Default

**Actual Result:**
Screen reader is not announcing the expanded/collapsed state for the dropdown controls present in the header section in Windows.

**

https://github.com/user-attachments/assets/506a90c8-f746-40aa-8a8a-3fd730f1f050
<img width="960" height="624" alt="Image" src="https://github.com/user-attachments/assets/a73adcb0-b945-408e-b021-7d2ac66dc24d" />

**- Narrator is not announcing the name property for the expand/collapse control in the Grid

### What was the expected behaviour?

The screen reader should announce the expanded or collapsed state of dropdown controls in the header section on Windows. When a dropdown is expanded or collapsed. It should announce it as for ex: Categories Expanded'.
Ensure that the dropdown should close using ESC key and the screen reader should announce as collapsed once the dropdown is closed.

### Was it tested on latest react-native-navigation?

- [x] I have tested this issue on the latest react-native-navigation release and it still reproduces.

### Help us reproduce this issue!

**Repro steps:**

1. Turn on Narrator.
2. Open URL:  https://www.wix.com/blog in the latest edge browser.
3. Press tab key from top of the page to move to the dropdown controls like Categories and resources present in the header section.
4. Verify whether the screen reader is announcing the expanded/collapsed state for the Categories and resources controls or not.

### In what environment did this happen?

Wix Version: WiX v5.0.0
.NET 6
HeatWave v1.0.3 includes

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by markdevocht** [comment]
@msftedad I think this is not the right place for this ticket. You added it to the React-Native-Navigation project.
